src/stories/Recollect.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { checkA11y } from '@storybook/addon-a11y';
+import { IntlProvider } from 'react-intl';
+
+
+import Recollect from 'js/modules/Recollect';
+
+
+storiesOf('Recollect', module)
+  .addDecorator(checkA11y)
+  .add('recollect', () => (
+    <IntlProvider locale="en">
+      <div className="wrapper container-fluid">
```

Comment:
Would it just make sense to have a global wrapper container-fluid decorator? or maybe just a decorator method we can use for this as needed.

Reply:
thought about this but we don't want it for Header, Footer, PageBanner, LanguageSelect and maybe a few others.